pkg/apis/rollouts/v1alpha1/generated.proto

```diff
@@ -1404,22 +1408,28 @@ message WavefrontMetric {
 }
 
 message WebMetric {
+  // Method is the method of the web metric (empty defaults to GET)
+  optional string method = 1;
+
   // URL is the address of the web metric
-  optional string url = 1;
+  optional string url = 2;
 
   // +patchMergeKey=key
   // +patchStrategy=merge
   // Headers are optional HTTP headers to use in the request
-  repeated WebMetricHeader headers = 2;
+  repeated WebMetricHeader headers = 3;
+
+  // Body is the body of the we metric (must be POST/PUT)
```

Comment:
@khhirani are these changes part of this PR? They look like they are from another PR? Or am I missing something?

Reply:
I'm not sure where this change is from. It appeared when I ran `make codegen`.